### PR TITLE
fix(Style guide): Embed images

### PR DESCRIPTION
**Pull request by barbnewrelic**
Noticed a stray backtick in an example and decided to copyedit as long as I was here.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx [hunk 1]
```diff
@@ -33,32 +32,32 @@ Here are some things to keep in mind when you're creating an image:
...
 
 To add an image from scratch:
-1. Find the images directory for your doc. For example, if your docs lives in `docs/style-guide/processes-procdures` you would use the `docs/style-guide/processes-procdures/images` directory. If the `images` directory does not exist, create one in [VScode](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_create-or-open-a-file) or Finder.
-2. Place your image in the `images` directory. Give the image a descriptive file name: `fso-ui-overview.png` is much better than `123-go-dawgs.png`. 
-3. Follow the [steps below](#embed) to embed the image in a doc. 
+
+1. Find the images directory for your doc. For example, if your doc lives in `docs/style-guide/processes-procedures`, use the `docs/style-guide/processes-procedures/images` directory. If the `images` directory does not exist, create one in [VScode](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_create-or-open-a-file) or Finder.
+2. Place your image in the `images` directory. Give the image a descriptive file name; for example, `fso-ui-overview.png` is much better than `123-go-dawgs.png`.
```
Comment (dbarnesbrownNR) [inline src/content/docs/style-guide/processes-procedures/embed-images.mdx:40]: Might want to call out that `images` where it names files always needs to be lowercase and plural as this has been a point of confusion in the past.

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx [hunk 2]
```diff
@@ -136,21 +142,22 @@ If your icon appears as an attribute inside another tag, prefix it with `icon` a
 If your icon appears inside running text, use the `<Icon>` component. Here are some examples:
 
 * Feather: `<Icon name="fe-database" />`
-* New Relic:`<Icon name="nr-tdp" />` 
+* New Relic: `<Icon name="nr-tdp" />` 
 * Logos: `<Icon name="logo-apple" />`
 
 ### Install new Feather icons
```
Comment (dbarnesbrownNR) [inline src/content/docs/style-guide/processes-procedures/embed-images.mdx:149]: Add `[#feather-icons]` to header?

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx [hunk 3]
```diff
@@ -93,29 +100,28 @@ Here's an example of the HTML for a fixed width, block level image:
...
-* [Docs site Feather icons](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104): These are the Feather icons available in the docs project but are not included in the Gatsby theme.
+* [Feather.js for `docs-website` in GitHub](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104): These are the Feather icons available in the `docs-website` project but are not included in the Gatsby theme.
+* [Gatsby theme for `docs-website` in GitHub](https://github.com/newrelic/docs-website/tree/develop/src/%40newrelic/gatsby-theme-newrelic/icons): This is a subset of Feather, New Relic, and product logo icons that are available across the developer and docs sites.
+
+You can use any icons you need in the Gatsby theme or Feather. Currently these locations have separate, non-overlapping buckets of icons.
 
 ### Insert icons as tag attributes
```
Comment (dbarnesbrownNR) [inline src/content/docs/style-guide/processes-procedures/embed-images.mdx:127]: Add `[#tag-attributes]` to header?

**Review by dbarnesbrownNR — APPROVED** [review]
Looks good but made a couple of comments.

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx (hunk 1 above)
Comment (barbnewrelic) [reply src/content/docs/style-guide/processes-procedures/embed-images.mdx:40]: Great catch, thanks!

**Pull request merged**